src/nccl_ofi_rdma.cpp

```diff
@@ -2455,6 +2492,27 @@ static int finish_connect(nccl_net_ofi_rdma_send_comm_t *s_comm)
 	return ret;
 }
 
+static inline uint64_t *get_flush_buffer_for_rail(void *ptr, uint16_t rail_id){
+	return static_cast<uint64_t*>(static_cast<void*>(static_cast<char*>(ptr) + (cpu_cache_line_size * rail_id)));
```

Comment:
ugh; sure we can't do this in some semi-sane way?  Like have `ptr` be a struct that contains two entries:

```
struct {
   uint64_t flag;
  char *padding[cpu_cache_line_size - sizeof(uint64_t)];
}
```
and then you can have an array of those of length MAX_NUM_RAILS and you don't need this line that makes my head hurt.